src/content/guides/production.md

```diff
@@ -236,9 +237,33 @@ __src/index.js__
   document.body.appendChild(component());
 ```
 
+## Module Concatenation
+
+By default, webpack wraps each module into a function. This helps helps to isolate them, but brings additional overhead. You can try reducing this overhead by enabling [`ModuleConcatenationPlugin`](/plugins/module-concatenation-plugin) which, where possible, merges multiple modules together in a single module wrapper:
+
+__webpack.prod.js__
+
+```diff
++ const webpack = require('webpack');
+  const merge = require('webpack-merge');
+  const common = require('./webpack.common.js');
+
+  module.exports = merge(common, {
+    plugins: [
++     new webpack.optimize.ModuleConcatenationPlugin()
+    ]
+  })
+```
+
+Read more about module concatenation [in the plugin docs](/plugins/module-concatenation-plugin).
 
 ## CLI Alternatives
 
-Some of what has been described above is also achievable via the command line. For example, the `--optimize-minimize` flag will include the `UglifyJSPlugin` behind the scenes. The `--define process.env.NODE_ENV="'production'"` will do the same for the `DefinePlugin` instance described above. And, `webpack -p` will automatically invoke both those flags and thus the plugins to be included.
+Some of what has been described above is also achievable via the command line:
+* the `--optimize-minimize` flag will include the `UglifyJSPlugin` behind the scenes;
+* the `--define process.env.NODE_ENV="'production'"` will do the same for the `DefinePlugin` instance described above;
+* the `--concatenate-modules` flag will enable the `ModuleConcatenationPlugin`.
```

Comment:
To fix the build errors add a line break between 262/263 and change the list style to `-`. I also think you can drop the leading `the` in these and simplify like so:

> - `--optimize-minimize` will include the `UglifyJSPlugin` behind the scenes;
> - `--define process.env.NODE_ENV="'production'"` will allow dependencies to key off `process.env.NODE_ENV`;
> - `--concatenate-modules` will include the `ModuleConcatenationPlugin`.